docs/concepts/browsers-used-by-office-web-add-ins.md

```diff
@@ -58,13 +58,13 @@ To determine whether Office 2016 or Office 2019 is retail or volume-licensed, us
 | Office 2013 | Doesn't matter | Doesn't matter | Internet Explorer 11 |
 | Office 2016 | Volume-licensed | Doesn't matter | Internet Explorer 11 |
 | Office 2019 | Volume-licensed | Doesn't matter | Internet Explorer 11 |
-| Office 2016 to Office 2019 | Retail | No | Microsoft Edge<sup>1, 2</sup> with original WebView (EdgeHTML)</br>If Edge is not installed, Internet Explorer 11 is used. |
+| Office 2016 to Office 2019 | Retail | No | Microsoft Edge Legacy (EdgeHTML)<sup>1, 2</sup></br>If Edge isn't installed, Internet Explorer 11 is used. |
```

Comment:
You already caveat (EdgeHTML) earlier in the article. Do you need to do it again here, or would "Microsoft Edge Legacy" be sufficient? If you do change it here, please change it later in the article as well.

Reply:
The EdgeHTML reference first appears in the Important block. My thought is that if the user is directly linked to the "Perpetual versions" section or somehow skips the Important block, they would have another way to differentiate between the old and new Edge versions through that EdgeHTML reference.